src/api/touringcar/messages/index.ts

```diff
@@ -0,0 +1,64 @@
+import { Feature, FeatureCollection, Point } from 'geojson'
+import { DateHumanReadable_Year_Month_Day } from 'shared/utils/dateTime'
+
+// Unlike other Touringcar API endpoints the messages endpoint
+// sits on the "Bereikbaarheid API" and NOT on api.data.amsterdam.nl!
+import { api } from './api_TEMP'
+
+interface TouringcarMessage extends Feature {
+  geometry: Point
+  properties: {
+    title: string
+    body: string
+    advice: string
+    title_en: string
```

Comment:
Waarschijnlijk prima voor nu, maar ik zou verwachten dat er een nettere methode is om verschillende talen te ondersteunen, ipv ze allemaal op deze manier los op te nemen in de code? Ik heb er zelf geen ervaring mee hoor, maar als je nu een taal toevoegt, moet je (o.a.?) deze code ook aanpassen.